package.json

```diff
@@ -7,10 +7,10 @@
     "@tensorflow-models/coco-ssd": "^2.2.2",
     "@tensorflow-models/posenet": "^2.2.2",
     "@tensorflow/tfjs": "^3.8.0",
-    "@tensorflow/tfjs-backend-cpu": "^3.9.0",
-    "@tensorflow/tfjs-backend-webgl": "^3.9.0",
-    "@tensorflow/tfjs-core": "^3.9.0",
-    "@tensorflow/tfjs-node": "^3.9.0",
+    "@tensorflow/tfjs-backend-cpu": "^3.8.0",
```

Comment:
I noticed issues with different versions of tfjs, the 3.9.0 on other hand led to some issues. Could be that my project wasn't rebuild correctly.